_data/site_menu.yml

```diff
@@ -31,47 +31,58 @@ installation:
       path: "/docs/reference/latest/installing_on_scvmm/index.html"
       desc: "How to install and configure the ManageIQ Management Engine appliance on Microsoft System Center Virtual Machine Manager (SCVMM)"
 
-appliances:
-  title: Appliances
+upgrading:
+  title: Upgrading
   children:
-    - title: Backup and Restore
-      path: "/docs/reference/latest/appliances/backup_and_restore.html"
-      desc: "How to backup and restore appliances"
     - title: Upgrading
       path: "/docs/reference/latest/appliances/upgrading.html"
       desc: "How to upgrade appliances"
 
-configuration:
-  title: Configuration
-  children:
-    - title: Deployment Planning Guide
-      path: "/docs/reference/latest/deployment_planning_guide/index.html"
-      desc: "ManageIQ Deployment Planning Guide"
-    - title: General Configuration
-      path: "/docs/reference/latest/general_configuration/index.html"
-      desc: "ManageIQ General Configuration"
-    - title: High Availability Guide
-      path: "/docs/reference/latest/high_availability_guide/index.html"
-      desc: "ManageIQ High Availability Guide"
-    - title: Appliance Hardening Guide
-      path: "/docs/reference/latest/appliance_hardening_guide/index.html"
-      desc: "ManageIQ Appliance Hardening Guide"
-
 administration:
   title: Administration
+  children:
+    - title: Provisioning Virtual Machines and Hosts
+      path: "/docs/reference/latest/provisioning_virtual_machines_and_hosts/index.html"
+      desc: "Provisioning, workload management, and orchestration in ManageIQ"
+    - title: Service Provisioning Using an Orchestration Template
+      path: "/docs/reference/latest/service_provisioning_using_an_orchestration_template/index.html"
+      desc: "How to create orchestration template to deploy services"
+    - title: Creating a Service in for Virtual Machine Provisioning
+      path: "/docs/reference/latest/creating_a_service_for_vm_provisioning/index.html"
+      desc: "How to create a service for virtual machine provisioning"
+    - title: Assigning a Custom Analysis Profile to a Virtual Machine
+      path: "/docs/reference/latest/assigning_custom_analysis_profile_to_vm/index.html"
+      desc: "How to assign custom analysis profile to a virtual machine"
```

Comment:
I don't know that most of these fall under "Administration", I do think administration for backup&restore or something like creating local users but these are all more related to managing inventory IMO.

Reply:
Good point.  I don't know how to word it but one is administration of the application and one is using the application.

Perhaps a high level "using" section with categories...
* Using
  * Authentication...
  * Automation
    * All of the provisioning, service, etc. 
    * Scripting actions ...
  * Policy
    * Monitoring, alerts, reporting...
    * Policies and Profiles guide
  * Manage Infra and Inv.
  * Managing Providers...
* Configuration
  * Assigning a Custom Analysis Profile...
  * Deployment planning guide
  * ...
* Administration
  * Appliance backup/restore
  * installion
  * upgrading
...

Reply:
Note, I would prefer we move things one at a time if we can't have a good visual before/after screenshots.  Ideally, each commit changes one thing and the PR changes aren't too much so it's easy to review and get merged.  Anything that "feels" like it's a separate change should probably be done in followup PRs so the high level reorganization can go in first.

Reply:
I can make changes to each big heading (section) in a single commit. With regards to Installing, Upgrading, It will make sense for it to be its own heading and not be under Administration.

Reply:
@jrafanie @agrare For the following, Should these go under `Configuration` or `Managing inventory`

- Provisioning Virtual Machines and Hosts
- Service Provisioning Using an Orchestration Template
- Creating a Service in for Virtual Machine Provisioning

Reply:
> * Provisioning Virtual Machines and Hosts
> * Service Provisioning Using an Orchestration Template
> * Creating a Service in for Virtual Machine Provisioning

I think they would go in using -> automation